crypto/dilithium/pqcrystals_dilithium_ref_common/symmetric-shake.c

```diff
@@ -1,28 +1,49 @@
 #include <stdint.h>
 #include "params.h"
 #include "symmetric.h"
-#include "fips202.h"
 
-void dilithium_shake128_stream_init(keccak_state *state, const uint8_t seed[SEEDBYTES], uint16_t nonce)
+/*************************************************
+* Name:        dilithium_shake128_stream_init
+*
+* Description: Initalise and absorb step of SHAKE128 specialized for ML-DSA
+*              context. Takes an uninitialized state |ctx| and initializes an
+*              output stream of SHAKE128(seed|nonce) using |seed| and |nonce|.
+*
+* Arguments:   - KECCAK1600_CTX *ctx: pointer to input/output Keccak state
+*              - const uint8_t seed[]: byte array with seed of length CRHBYTES
+*              - uint16_t nonce: 2-byte nonce
+*
+**************************************************/
+void dilithium_shake128_stream_init(KECCAK1600_CTX *ctx, const uint8_t seed[SEEDBYTES], uint16_t nonce)
```

Comment:
let's get rid of these two functions and do the work inline (they are called in one place only anyway).

Reply:
`dilithium_shake128_stream_init` and `dilithium_shake256_stream_init`

Reply:
Ok! `stream128_init` was called in two places within `poly.c`, but happy to inline! Done in [b934dd0](https://github.com/aws/aws-lc/pull/2001/commits/b934dd03956682a4ec79a2f764cb84e00403af3c)